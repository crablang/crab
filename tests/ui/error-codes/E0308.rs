#![feature(intrinsics)]
#![feature(crablangc_attrs)]

extern "crablang-intrinsic" {
    #[crablangc_safe_intrinsic]
    fn size_of<T>(); //~ ERROR E0308
}

fn main() {
}
