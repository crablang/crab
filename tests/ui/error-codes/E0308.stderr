error[E0308]: intrinsic has wrong type
  --> $DIR/E0308.rs:6:5
   |
LL |     fn size_of<T>();
   |     ^^^^^^^^^^^^^^^^ expected `()`, found `usize`
   |
   = note: expected fn pointer `extern "crablang-intrinsic" fn()`
              found fn pointer `extern "crablang-intrinsic" fn() -> usize`

error: aborting due to previous error

For more information about this error, try `crablangc --explain E0308`.
